setup.py

```diff
@@ -13,7 +13,7 @@
 setup(
     name="aiounifi",
     packages=["aiounifi"],
-    version="18",
+    version="19",
```

Comment:
Bumping version doesn't belong to a feature PR

```suggestion
    version="18",
```

Reply:
ah, my bad. fixed